web/views.py

```
@@ -2730,7 +2742,118 @@ def challenge_submit(request, week_number):
+    Display the leaderboard with highest scores first.
+    Also shows quiz results if the user just submitted the quiz.
+    Args:
+        request: HTTP request object
+        quiz_id: ID of the quiz to show leaderboard for
+    Returns:
+        Rendered leaderboard template
+    """
+    try:
+        quiz = get_object_or_404(Quiz, id=quiz_id)
+        # Retrieve all submissions for this quiz, ordered by highest score
+        submissions = QuizSubmission.objects.filter(quiz=quiz).order_by("-score", "submitted_at")
```

Ruff (0.8.2)

Local variable `submissions` is assigned to but never used

Remove assignment to unused variable `submissions`

(F841)